src/lsst/rsp/startup/services/labrunner.py

```diff
@@ -373,7 +480,7 @@ def _merge_pgpass(self) -> None:
             connection, passwd = line.rsplit(":", maxsplit=1)
             config[connection] = passwd.rstrip()
         # Update config from container-supplied one
-        ro_pgpass = Path(self._env["ORIG_PGPASSFILE"])
+        ro_pgpass = Path(self._env.get("ORIG_PGPASSFILE", ""))
```

Comment:
Here, I think you're just going to get weird exceptions that will be hard to debug if you continue with a path of `""`.